### PR TITLE
{numlib}[foss/2019b] Armadillo v9.860.1

### PR DESCRIPTION
**Pull request by Falkor**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-084 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.7.4
See https://gist.github.com/35a6e8d845afeeecdd384a2c2eade14c for a full test report.

File: easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+]
+
+builddependencies = [('CMake', '3.15.3')]
+
+sanity_check_paths = {
+    'files': ["lib/libarmadillo.%s" % SHLIB_EXT,
+              'share/Armadillo/CMake/ArmadilloConfig.cmake'],
+    'dirs': []
+}
+
```
Comment (lexming) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb:27]: The `sanity_check_paths` do not seem necessary. The easyblock of Armadillo will already take care of them.
```suggestion
```

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-003 - Linux centos linux 7.7.1908, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/b470531c59af4a63f4fd084c9eb64e3f for a full test report.

File: easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb (hunk 1 above)
Comment (Falkor) [reply easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb]: done in c2fdecd

**Comment by Falkor** [comment]
I think I addressed the changes or did I missed something?

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
node380.hydra.os - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/9994fb1f4d56280f5495dddc8c027c54 for a full test report.

File: easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+homepage = 'https://arma.sourceforge.net/'
+description = """Armadillo is an open-source C++ linear algebra library (matrix maths) aiming towards
+ a good balance between speed and ease of use. Integer, floating point and complex numbers are supported,
+ as well as a subset of trigonometric and statistics functions."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+source_urls = ['https://sourceforge.net/projects/arma/files']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['1603888ab73b7f0588df1a37a464436eb0ff6b1372a9962ee1424b4329f165a9']
```
Comment (lexming) [inline easybuild/easyconfigs/a/Armadillo/Armadillo-9.860.1-foss-2019b.eb:13]:
```suggestion
patches = [
    'Armadillo-9.600.5_no_mkl.patch'
]
checksums = [
    '1603888ab73b7f0588df1a37a464436eb0ff6b1372a9962ee1424b4329f165a9',  # src
    '8288e769ceaf06da9f6870f616272c354577b1dd6f07e923fed43b22bae131b6',  # Armadillo-9.600.5_no_mkl.patch
]
```

**Comment by Falkor** [comment]
Latest stable release is now 900.1 -- I updated the easyconfigs accordingly.

**Comment by Falkor** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-035 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/05432970e54c153fc043d4a0673e9b92 for a full test report.

**Comment by lexming** [comment]
@Falkor we already have `Armadillo-9.900.1-foss-2019b.eb` in the develop branch

**Comment by lexming** [comment]
@Falkor since `Armadillo-9.900.1-foss-2019b.eb` is already in the repository, may we close this PR?

**Comment by Falkor** [comment]
Yes sure ;)

Sent with my iPhone

Le 8 juil. 2020 à 16:16, Alex Domingo <notifications@github.com> a écrit :

﻿

@Falkor<https://github.com/Falkor> since Armadillo-9.900.1-foss-2019b.eb is already in the repository, may we close this PR?

**Pull request closed without merging**